appointments/views.py

```diff
@@ -114,6 +114,7 @@ def edit_position(request,position_id):
     
     if position_id == '':
       p = Position()
+      print "what???" + p
```

Comment:
@wjones127 any idea w.r.t this?????
if i don't use a print statement there's an error...